libopenage/util/compiler.cpp

```diff
@@ -33,6 +40,9 @@ std::string addr_to_string(const void *addr) {
 
 
 std::string symbol_name(const void *addr, bool require_exact_addr, bool no_pure_addrs) {
+#ifdef _MSC_VER
+	return no_pure_addrs ? "" : addr_to_string(addr);
```

Comment:
Similar to above (although, clearly I forgot to add a `TODO` - I'll do that).
There's [`SymFromAddr`](https://msdn.microsoft.com/en-us/library/windows/desktop/ms681323(v=vs.85).aspx), which has the same pitfalls as the one mentioned above.
This problem is actually discussed in #480, and I would like to know their opinions on using this approach.